### PR TITLE
fix various APU logic issues

### PR DESCRIPTION
**Pull request by UncleClapton**
## Changes introduced
* Change initial state of some local vars so the APU page reinitializes correctly on dev reload.
* Improve APU Master switching logic
* Improve APU Bleed switching logic
* Tie APU GEN visibility to master switch and APU RPM instead of it relying on flap logic
  - When the master switch is on, the APU GEN info properly becomes visible. A bug previously prevented this from happening.
  - When the master switch is off, and RPM is <= 94, the APU GEN info disappears. (This is modeled from the FSLabs a320)
* Fix bug caused by typo in EGT calculation

This was developed along side @AdenFlorian's changes. Some of the code overlaps but we've discussed this and are okay with dealing with the merge conflicts caused by which ever one gets merged first.

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 1]
```diff
@@ -76,89 +81,91 @@ var A320_Neo_LowerECAM_APU;
...
-                else{
-                    this.APUBleedTimer = -1;
+                    SimVar.SetSimVarValue("L:APU_BLEED_PRESSURE","PSI",Math.round(35-this.APUBleedTimer));
                 }
+
                 this.APUAvail.setAttribute("visibility", "visible");
-                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) this.APUGenAvailArrow.setAttribute("visibility", "visible");
-                else this.APUGenAvailArrow.setAttribute("visibility", "hidden");
+
+                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) {
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:139]: always use `==` for comparing simvar values

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 2]
```diff
@@ -258,7 +265,7 @@ var A320_Neo_LowerECAM_APU;
                 if (n < 10) {
                     return 10;
                 } else if(n <14){
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:267]: That formatting should be fixed while you're at it

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 3]
```diff
@@ -258,7 +265,7 @@ var A320_Neo_LowerECAM_APU;
...
-                    reuturn ((90/6*n)- 140);
+                    return ((90/6*n)- 140);
                 } else if (n < 20) {
                     return ((215/4*n)-760);
                 } else if(n < 32){
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:271]: Ditto

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js (hunk 1 above)
Comment (Benjozork) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:139]: actually, we are not sure

Comment (UncleClapton) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:139]: *Citation needed*

(we're gonna double check this is actually something we need to worry about)

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js (hunk 3 above)
Comment (UncleClapton) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:271]: done

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js (hunk 2 above)
Comment (UncleClapton) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js]: done

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js (hunk 1 above)
Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:139]: I've === to be fine, but as always it should be tested anyways to make sure its working.

**Comment by UncleClapton** [comment]
This is ready for review / merge again.

**Pull request merged**